### ISSUE TITLE
Broken links in readme

### ISSUE THREAD

**Issue by 304NotModified** [Pablissimo/SonarTsPlugin#151]
Thanks for the great plugin!

These links are broken in the readme:

![image](https://user-images.githubusercontent.com/5808377/27658335-fb2b441e-5c4f-11e7-825e-87d96aa7f2c2.png)

![image](https://user-images.githubusercontent.com/5808377/27658343-fe20fdbc-5c4f-11e7-93d3-9baad24c6807.png)

**Comment by Pablissimo** [comment]
Yeah I've been having trouble keeping them up without taking down the whole VM they're currently hosted on (made the mistake of upgrading the hosting environment). Will try to sort them out, and get a 6.4 site running for regression testing.
